### PR TITLE
Added a libyui-terminal helper script (boo#937026).

### PR DESCRIPTION
**Pull request by mvidner**
https://bugzilla.opensuse.org/show_bug.cgi?id=937026

### CODE REVIEW

File: src/libyui-terminal [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+EXIT=`mktemp`
+OUT=`mktemp`
+
+echo "Using `basename $0` to run \"$@\""
+# escape for embedding to "sh -c"
+quoted=$(printf "%q " "$@")
+# -D -m   This also starts screen in "detached" mode, but doesn't fork
+#         a new process. The command exits if the session
+#         terminates.
+screen -D -m sh -c "$quoted > $OUT 2>&1; echo \$? > $EXIT"
```
Comment (jreidinger) [inline src/libyui-terminal]: I do not see in spec dependency on screen. How it is handled?

Comment (mvidner) [inline src/libyui-terminal]: Yes, that's a weak point. If you use this you need to install screen on your own. Ideally I would add it to the "testing" group of a gemfile, but this is not a gem. I thought this was a good package to place the script in but the packages should not require screen.
Any suggestions to improve the situation?

Comment (ancorgs) [inline src/libyui-terminal]: Should we modify the libyui-terminal script so it shows a sensible message if screen is not found?

Comment (jreidinger) [inline src/libyui-terminal]: agreed, at least make it prepared for situation, so write error message that this helper script require screen.

Comment (mvidner) [inline src/libyui-terminal]: Fair concern, and I think that out of the box it behaves sensibly:
```console
$ LANG=C ./libyui-terminal rake test:unit
Using libyui-terminal to run "rake test:unit"
./libyui-terminal: line 36: screen: command not found
libyui-terminal finished
```

**Comment by ancorgs** [comment]
Agreed that the default error message already looks good enough. So LGTM.

**Pull request merged**